meta/saisanitycheck.c

```diff
@@ -332,6 +332,26 @@ bool sai_metadata_is_acl_field_or_action(
     return false;
 }
 
+bool sai_metadata_is_tlv_or_segment(
```

Comment:
Please remove this merhod, no default value od only for acl, on Your proposal add default value as empty od internautów od list is not empty And thenyou font need thisncheck, please see other attribute how lista are using default value